examples/kieservers/basic-auth-vault.yml

```
@@ -0,0 +1,10 @@
+quarkus:
+  file:
+    vault:
+      provider:
+        pim:
```

not sure about this, but as per `credentials-provider` property defined above, shouldn't we need to include another entry within `pim` containing `kieserver`?:

```suggestion
        pim:
           kieserver:
```

---

I've used `kieserver1` key instead.